lib/ClangImporter/SwiftLookupTable.cpp

```diff
@@ -195,6 +195,13 @@ translateDeclToContext(clang::NamedDecl *decl) {
     return None;
   }
 
+  // Namespace declaration.
+  if (auto tag = dyn_cast<clang::NamespaceDecl>(decl)) {
+    if (tag->getIdentifier())
+      return std::make_pair(SwiftLookupTable::ContextKind::Tag, tag->getName());
```

Comment:
I second- and third-guessed the decision to use "tag" here since C++ namespaces aren't in the "tag" identifier namespace, so maybe we should come up with a better name, but I guess it's okay.

Reply:
Whoops! I just copy pasted from the tag-decl version to save on typing. No underlying wisdom to the choice.